mimic/rest/loadbalancer_api.py

```diff
@@ -54,10 +55,128 @@ def resource_for_region(self, region, uri_prefix, session_store):
         Get an :obj:`twisted.web.iweb.IResource` for the given URI prefix;
         implement :obj:`IAPIMock`.
         """
+        self.session_store = session_store
         lb_region = LoadBalancerRegion(self, uri_prefix, session_store,
                                        region)
         return lb_region.app.resource()
 
+    def _get_session(self, session_store, tenant_id):
+        """
+        Retrieve or create a new LoadBalancer session from a given tenant identifier
+        and :obj:`SessionStore`.
+
+        For use with ``data_for_api``.
```

Comment:
Given that this *calls* `data_for_api`, it is a bit misleading to me to document it this way; it makes it sound like the caller should be doing something with `_get_session` and then calling `data_for_api_ themselves, which is backwards; they should let this function call it.